blaze/mahimahi/server/server.py

```diff
@@ -16,6 +20,45 @@
With.apply(this,arguments)},old_insertAdjacentHTML=Element.prototype.insertAdjacentHTML,Element.prototype.insertAdjacentHTML=function(){let t=arguments[1]+"",e=getURLfromString(t);if(null!=e)StackTrace.get().then(t=>{stackInterceptCallback(t,e)}).catch(t=>{stackInterceptErrBack(t)});else{let e=htmlToElement(t=arguments[1]);if(void 0!==e.href)StackTrace.get().then(t=>{stackInterceptCallback(t,e.href)}).catch(t=>{stackInterceptErrBack(t)});else if(void 0!==e.src)StackTrace.get().then(t=>{stackInterceptCallback(t,e.src)}).catch(t=>{stackInterceptErrBack(t)});else{let e=t;console.log(e),void 0!==e.href&&StackTrace.get().then(t=>{stackInterceptCallback(t,e.href)}).catch(t=>{stackInterceptErrBack(t)}),void 0!==e.src&&StackTrace.get().then(t=>{stackInterceptCallback(t,e.src)}).catch(t=>{stackInterceptErrBack(t)})}}old_insertAdjacentHTML.apply(this,arguments)},old_insertAdjacentText=Element.prototype.insertAdjacentText,Element.prototype.insertAdjacentText=function(){let t=arguments[1]+"",e=getURLfromString(t);if(null!=e)StackTrace.get().then(t=>{stackInterceptCallback(t,e)}).catch(t=>{stackInterceptErrBack(t)});else{let e=htmlToElement(t=arguments[1]);if(void 0!==e.href)StackTrace.get().then(t=>{stackInterceptCallback(t,e.href)}).catch(t=>{stackInterceptErrBack(t)});else if(void 0!==e.src)StackTrace.get().then(t=>{stackInterceptCallback(t,e.src)}).catch(t=>{stackInterceptErrBack(t)});else{let e=t;console.log(e),void 0!==e.href&&StackTrace.get().then(t=>{stackInterceptCallback(t,e.href)}).catch(t=>{stackInterceptErrBack(t)}),void 0!==e.src&&StackTrace.get().then(t=>{stackInterceptCallback(t,e.src)}).catch(t=>{stackInterceptErrBack(t)})}}old_insertAdjacentText.apply(this,arguments)},old_insertAdjacentElement=Element.prototype.insertAdjacentElement,Element.prototype.insertAdjacentElement=function(){let t=arguments[1]+"",e=getURLfromString(t);if(null!=e)StackTrace.get().then(t=>{stackInterceptCallback(t,e)}).catch(t=>{stackInterceptErrBack(t)});else{let e=htmlToElement(t=arguments[1]);if(void 0!==e.href)StackTrace.get().then(t=>{stackInterceptCallback(t,e.href)}).catch(t=>{stackInterceptErrBack(t)});else if(void 0!==e.src)StackTrace.get().then(t=>{stackInterceptCallback(t,e.src)}).catch(t=>{stackInterceptErrBack(t)});else{let e=t;console.log(e),void 0!==e.href&&StackTrace.get().then(t=>{stackInterceptCallback(t,e.href)}).catch(t=>{stackInterceptErrBack(t)}),void 0!==e.src&&StackTrace.get().then(t=>{stackInterceptCallback(t,e.src)}).catch(t=>{stackInterceptErrBack(t)})}}old_insertAdjacentElement.apply(this,arguments)},old_replaceChild=Element.prototype.replaceChild,Element.prototype.replaceChild=function(){console.log("element.replaceChild intercepted.");let t=arguments[0];void 0!==t.href&&StackTrace.get().then(e=>{stackInterceptCallback(e,t.href)}).catch(t=>{stackInterceptErrBack(t)}),void 0!==t.src&&StackTrace.get().then(e=>{stackInterceptCallback(e,t.src)}).catch(t=>{stackInterceptErrBack(t)}),old_replaceChild.apply(this,arguments)};var imagesInViewPort=[]
```

Comment:
Is this string the same as `blaze/mahimahi/server/critical_requests_extract.js`? If so read the file here

Reply:
critical_requests_extract.js contains the unminified version. the string above is the minified version. i added the unminified file in case we wanted to make some changes to the script.